src/ExpressionEvaluator/Core/Source/ResultProvider/SystemHelpers.cs

```diff
@@ -0,0 +1,45 @@
+﻿// A few dependencies from System.dll, which we want to avoid when compiling against Desktop Framework 2.0.
+namespace System.Linq
+{
+    internal class Dummy { }
```

Comment:
This is just needed since we auto-import System.Linq in all VB projects. Maybe we shouldn't but that would be a bigger change.